### ISSUE TITLE
fatih: seperation-of-concerns, 3 weeks

### ISSUE THREAD

**Issue by Fatihayoz** [HackYourFutureBelgium/class-13-14#273]
> [module repository](https://github.com/Fatihayoz/separation-of-concerns)

## Study Plan

[Here :computer: :books: :bookmark_tabs:](https://github.com/Fatihayoz/separation-of-concerns/projects/1)

## Week 1

- [ ] started the exercises
- [ ] [project issue](https://github.com/HackYourFutureBelgium/class-13-14/issues/268)

**Comment by Fatihayoz** [comment]
study plan is set for seperation-of-concerns module

**Comment by Fatihayoz** [comment]
check-in for W1 edited.

**Comment by Lizhen2910** [comment]
@Fatihayoz yes time management is a challenge to all of us. Let me know if you need help with that.

**Comment by Fatihayoz** [comment]
@Lizhen2910 thank you for your proposal. I had also an intensive week in Dutch class. I'm trying to fix it. :nerd_face:
